app/lib/resources.server.ts

```diff
@@ -96,26 +95,35 @@ declare global {
 
 let NO_CACHE = env.NO_CACHE;
 
-global.resourceReadmeCache ??= new LRUCache<string, string>({
+global.resourceReadmeCache ??= new LRUCache<string, string, CacheContext>({
   max: 300,
   ttl: NO_CACHE ? 1 : 1000 * 60 * 5, // 5 minutes
   allowStale: !NO_CACHE,
   noDeleteOnFetchRejection: true,
   fetchMethod: fetchReadme,
 });
 
-async function fetchReadme(repo: string): Promise<string> {
-  let md = await getRepoContent(repo, "main", "README.md");
-  if (md === null) {
-    throw Error(`Could not find README in ${repo}`);
-  }
+async function fetchReadme(
+  key: string,
+  _staleValue: string | undefined,
+  { context }: LRUCache.FetchOptionsWithContext<string, string, CacheContext>,
+): Promise<string> {
+  let [owner, repo] = key.split("/");
+  let contents = await context.octokit.rest.repos.getReadme({
+    owner,
+    repo,
+    mediaType: { format: "base64" },
```

Comment:
I'm just now realizing that we can just make it `format: "raw"` 🤦. I feel like that makes sense, what do you think? Pretty sure raw will also be smaller in size and less work than getting a base64 encoding